lib/fetch.go

```diff
@@ -203,6 +222,11 @@ func (of *OCIFetcher) fetchManifest(u *URL) (*schema.ImageManifest, error) {
 		return nil, fmt.Errorf("unexpected http code: %d, URL: %s", res.StatusCode, req.URL)
 	}
 
+	digest := res.Header.Get("Docker-Content-Digest")
```

Comment:
It would make sense to me for it to just be a hash of the manifest, but I think the last time I looked at it I couldn't get manifest hashes to match what the docker hub was claiming the image digests were.